File: src/main/kotlin/Java/Biblioteca/Biblioteca.java
package Java.Biblioteca;

import java.util.ArrayList;

public class Biblioteca {
    ArrayList<Libro> Libros;

    ArrayList<Usuario> Usuarios;

    public Biblioteca() {
        Libros = new ArrayList<Libro>();
        Usuarios = new ArrayList<Usuario>();
    }

    public String agregarLibro(Libro libro) {
        Libros.add(libro);
        return "Libro agregado";
    }

    public String agregarUsuario(Usuario usuario) {
        Usuarios.add(usuario);
        return "Usuario agregado";
    }

    public String prestarLibros(String ibs,Usuario usuario) {

        for (Libro libro : Libros) {
            if (libro.isbn == ibs){
                libro.prestado = true;
                usuario.Libros.add(libro);
                return "Libro prestado";
            }
        }

        return "Libro no prestado";
    }


}

File: src/main/kotlin/kotlinOPP/Biblioteca/Biblioteca.kt
package Java.Biblioteca

class Bibliotecak {
    var Libros: ArrayList<Librok> = ArrayList()

    var Usuarios: ArrayList<Usuariok> = ArrayList()

    fun agregarLibro(librok: Librok): String {
        Libros.add(librok)

        return "Libro agregado"
    }

    fun agregarUsuario(usuario: Usuariok): String {
        Usuarios.add(usuario)
        return "Usuario agregado"
    }

    fun prestarLibros(ibs: String, usuariok: Usuariok): String {
        for (librok in Libros) {
            if (librok.isbn === ibs) {
                librok.prestado = true
                usuariok.Libros.add(librok)
                return "Libro prestado"
            }
        }

        return "Libro no prestado"
    }
}